### ISSUE TITLE
Windows Compatibility

### ISSUE THREAD

**Issue by zQueal** [tldr-pages/tldr#392]
First, any version of Windows is shown to be `win32` even `x86_64`:

    npm ERR! node v0.12.2
    npm ERR! npm  v2.7.4
    npm ERR! code EBADPLATFORM
    npm ERR! notsup Unsupported
    npm ERR! notsup Not compatible with your operating system or architecture: tldr@1.2.1
    npm ERR! notsup Valid OS:    !win32
    npm ERR! notsup Valid Arch:  any
    npm ERR! notsup Actual OS:   win32
    npm ERR! notsup Actual Arch: ia32

But:

    λ systeminfo | find /I "System type"
    System Type:               x64-based PC

That aside, are their any plans for Windows compatible versions?

**Comment by igorshubovych** [comment]
I am working at the moment on Windows version

**Comment by rprieto** [comment]
Thanks @zQueal and @igorshubovych for working on it already! Remind me, what's the main blockers for Windows compatibility?

**Comment by kz** [comment]
Looking forward to Windows compatibility! I'm not sure whether this contributes to the issue, but it appears to be incompatible under Cygwin as well.

```
{ ~ }  » npm install -g tldr                                                                    ~
npm ERR! Windows_NT 6.3.9600
npm ERR! argv "C:\\Program Files\\nodejs\\node.exe" "C:\\Program Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "install" "-g" "tldr"
npm ERR! node v0.12.4
npm ERR! npm  v2.10.1
npm ERR! code EBADPLATFORM

npm ERR! notsup Unsupported
npm ERR! notsup Not compatible with your operating system or architecture: tldr@1.2.1
npm ERR! notsup Valid OS:    !win32
npm ERR! notsup Valid Arch:  any
npm ERR! notsup Actual OS:   win32
npm ERR! notsup Actual Arch: x64

npm ERR! Please include the following file with any support request:
npm ERR!     C:\Users\zhang_000\npm-debug.log
```

**Comment by mallowigi** [comment]
I have submitted a pull request for the tldr-node-client if someone is interested

**Comment by zQueal** [comment]
> It doesn't work on the regular command line neither on PowerShell yet.

I'll wait. Cygwin is the devil.

EDIT: Actually seems to work just fine in normal conditions.

λ tldr tar

  tar
  Archiving utility
  Optional compression with gzip / bzip

  - create an archive from files
    tar cf target.tar file1 file2 file3

  - create a gzipped archive
    tar czf target.tar.gz file1 file2 file3

  - extract an archive in a target folder
    tar xf source.tar -C folder

  - extract a gzipped archive in the current directory
    tar xzf source.tar.gz

  - extract a bzipped archive in the current directory
    tar xjf source.tar.bz2

  - create a compressed archive, using archive suffix to determine the compression program
    tar caf target.tar.xz file1 file2 file3

  - list the contents of a tar file
    tar tvf source.tar

**Comment by kz** [comment]
@zQueal I use Babun, which is a pre-configured Cygwin with oh-my-zsh. It's really nice and I highly recommend it.

**Comment by igorshubovych** [comment]
Please upgrade to tldr@1.3.0

**Comment by kz** [comment]
Cygwin installation now works: 
```
{ ~ }  » npm install -g tldr                                                                    ~
C:\Users\zhang_000\AppData\Roaming\npm\tldr -> C:\Users\zhang_000\AppData\Roaming\npm\node_modules\t                                                                                                 ldr\bin\tldr
tldr@1.3.0 C:\Users\zhang_000\AppData\Roaming\npm\node_modules\tldr
├── os-homedir@1.0.1
├── ms@0.7.1
├── deep-extend@0.4.0
├── colors@1.1.2
├── wrench@1.5.8
├── marked@0.3.5
├── commander@2.9.0 (graceful-readlink@1.0.1)
├── async@1.5.0
├── string@3.3.1
├── mkdirp@0.5.1 (minimist@0.0.8)
├── rimraf@2.5.0 (glob@6.0.3)
├── cpr@1.0.0 (graceful-fs@4.1.2, rimraf@2.4.5)
├── request@2.67.0 (is-typedarray@1.0.0, forever-agent@0.6.1, aws-sign2@0.6.0, caseless@0.11.0, stri                                                                                                 ngstream@0.0.5, oauth-sign@0.8.0, form-data@1.0.0-rc3, tunnel-agent@0.4.2, isstream@0.1.2, json-stri                                                                                                 ngify-safe@5.0.1, extend@3.0.0, node-uuid@1.4.7, combined-stream@1.0.5, qs@5.2.0, tough-cookie@2.2.1                                                                                                 , mime-types@2.1.8, bl@1.0.0, har-validator@2.0.3, http-signature@1.1.0, hawk@3.1.2)
├── unzip2@0.2.5 (setimmediate@1.0.4, pullstream@0.4.1, readable-stream@1.0.33, match-stream@0.0.2,                                                                                                  fstream@0.1.31, binary@0.3.0)
└── lodash@3.10.1
{ ~ }  » tldr                                                                                   ~
29 Dec 14:51:19 - Usage: tldr <command>
{ ~ }  » tldr tar                                                                             ~ 1
Page not found
Try updating with "tldr --update", or submit a pull request to:
https://github.com/tldr-pages/tldr
{ ~ }  » tldr --update                                                                        ~ 1
Updating...
Done
{ ~ }  » tldr tar                                                                               ~

  tar
  Archiving utility
  Optional compression with gzip / bzip

  - create an archive from files
    tar cf target.tar file1 file2 file3

  - create a gzipped archive
    tar czf target.tar.gz file1 file2 file3

  - extract an archive in a target folder
    tar xf source.tar -C folder

  - extract a gzipped archive in the current directory
    tar xzf source.tar.gz

  - extract a bzipped archive in the current directory
    tar xjf source.tar.bz2

  - create a compressed archive, using archive suffix to determine the compression program
    tar caf target.tar.xz file1 file2 file3

  - list the contents of a tar file
    tar tvf source.tar


{ ~ }  »    
```

**Comment by igorshubovych** [comment]
Please report a bug in tldr-node-client.

**Comment by igorshubovych** [comment]
Windows support is done in NodeJS client. As well as some other clients are working too.
